### PR TITLE
Add support for UAE

### LINKED ISSUE #6480: Add support for UAE
**As a** merchant from UAE
**I want to** be able to use WCPayments
**So that** I can sell online via my Woo store.

The following is a checklist to help with this:

- [ ] Add the country to supported_countries function
- [ ] Add the country to countryFeeStripeDocsSectionNumbers function
- [ ] Onboard on Stripe using a test account form the country (make sure to share it in PR for testers)
- [ ] Make a payment using a test card
- [ ] Make sure the currency is showing properly
- [ ] Turn on multiple payment methods and make a test payment with one
- [ ] Simulate a dispute using one of the test cards. Make sure the disputes are working properly (ie, showing in the list + currency is good)
- [ ] Make sure the deposits are working properly (ie, double check with Stripe Docs the minimum deposits amount, frequency, wait time...)

### PR DESCRIPTION
**Pull request by dpaun1985**
Fixes #6480 

#### Changes proposed in this Pull Request

Add fee support for United Arab Emirates

#### Testing instructions
* On server side checkout 3557-gh-Automattic/woocommerce-payments-server
* **Base fees**
* Go to Woo -> Settings -> General and set Store Country to United Arab Emirates and currency to AED
* Place 2 orders and pay it with an AE credit card and international credit card
* check the applied fees on the order transaction ( in admin, Payments->Transactions )

*

-------------------

- [ ] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [ ] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `6595` or branch name `add/6480-uae-fees` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&branches.woocommerce-payments=add/6480-uae-fees) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: 4c68e6bfae2a61e2bbc08d3bf91e3fd8e6632504
- Build time: 2023-08-04 18:00:18 UTC

_Note: the build is updated when a new commit is pushed to this PR._

**Review by anu-rock — APPROVED** [review]
LGTM 🙌

**Review by mordeth — APPROVED** [review]
LGTM! ✅

File: client/utils/account-fees.tsx [hunk 1]
```diff
@@ -22,6 +22,7 @@ const countryFeeStripeDocsBaseLink =
 const countryFeeStripeDocsBaseLinkNoCountry =
 	'https://woocommerce.com/document/payments/faq/fees';
 const countryFeeStripeDocsSectionNumbers: Record< string, string > = {
+	AE: 'arabemirates',
```
Comment (elazzabi) [inline client/utils/account-fees.tsx]: @aheckler fyi in case this doesn't match with what we are preparing for documentation

**Review by elazzabi — APPROVED** [review]
Looks good!

File: client/payment-methods/style.scss [hunk 2]
```diff
@@ -107,3 +107,16 @@
 		position: fixed;
 	}
 }
+
+// RTL - hide flags dropdown for phone input
+// @TODO find a way to make the dropdown working for LTR
```
Comment (elazzabi) [inline client/payment-methods/style.scss:112]: I think this should probably go to a separate issue to keep track of it

File: client/utils/account-fees.tsx (hunk 1 above)
Comment (aheckler) [reply client/utils/account-fees.tsx]: @elazzabi This probably needs to be `united-arab-emirates` as the docs link to their fees will be: 

`https://woocommerce.com/document/woopayments/fees-and-debits/fees/#united-arab-emirates`

Comment (elazzabi) [reply client/utils/account-fees.tsx]: Fixed!

**Comment by elazzabi** [comment]
Fixed the phone input element here.

Before https://github.com/Automattic/woocommerce-payments/pull/6595/commits/f51f6545fc574eae47f3ca81796d6f129657c0b4:

The country code and flag were removed completely

https://github.com/Automattic/woocommerce-payments/assets/12542046/14701919-a419-4bec-979a-97035d2f85d4

After https://github.com/Automattic/woocommerce-payments/pull/6595/commits/f51f6545fc574eae47f3ca81796d6f129657c0b4:

At least we can see the country flag and code (only dropdown triangle is removed)


https://github.com/Automattic/woocommerce-payments/assets/12542046/567d3acb-0bbb-4d8c-810b-3dd1fe3dc2c6

**Pull request merged**